templates/pricing/desktop.html

```diff
@@ -10,60 +10,57 @@
-      <div class="col-8">
+  <section class="p-section--hero">
+    <div class="row--50-50">
+      <div class="col">
         <h1>
           Ubuntu Desktop
-          <br />
+          <br class="u-hide--small" />
           Enterprise Services
         </h1>
+      </div>
+      <div class="col">
         <p>Day zero support for Ubuntu Desktop deployment, compliance and custom configuration.</p>
-        <a class="p-button"
-           href="https://assets.ubuntu.com/v1/a48fc9cd-Ubuntu%20Desktop%20DS%2021.06.23.pdf">Download the datasheet</a>
-        <a href="/desktop/contact-us" class="p-button--positive js-invoke-modal">Get in touch</a>
+        <div class="p-cta-block">
+          <a class="p-button"
+             href="https://assets.ubuntu.com/v1/a48fc9cd-Ubuntu%20Desktop%20DS%2021.06.23.pdf">Download the datasheet</a>
+          <a href="/desktop/contact-us" class="p-button--positive js-invoke-modal">Get in touch</a>
+        </div>
       </div>
     </div>
   </section>
 
-  <section class="p-strip--light">
-    <div class="u-fixed-width">
-      <h2>Get your team up and running with Ubuntu</h2>
-      <p>
-        Ubuntu Desktop Enterprise Services are designed to help your developers get up and running on Ubuntu as quickly as possible. Canonical engineers partner with you to deliver a solution tailored to your needs, from custom images and Snap Store proxies to Active Directory integration and fleet management through Landscape.
-      </p>
-    </div>
-    <div class="row">
-      <div class="col-6 p-card">
-        <h2 class="p-heading--4">Deployment Steps</h2>
-        <ol class="p-stepped-list">
-          <li class="p-stepped-list__item u-no-padding--bottom">
-            <p class="p-stepped-list__title">Requirements gathering</p>
-          </li>
-          <li class="p-stepped-list__item u-no-padding--bottom">
-            <p class="p-stepped-list__title">Design documentation and deployment guide</p>
-          </li>
-          <li class="p-stepped-list__item u-no-padding--bottom">
-            <p class="p-stepped-list__title">Custom image creation</p>
-          </li>
-          <li class="p-stepped-list__item u-no-padding--bottom">
-            <p class="p-stepped-list__title">Post-deployment automation</p>
-          </li>
-          <li class="p-stepped-list__item u-no-padding--bottom">
-            <p class="p-stepped-list__title">Active Directory integration</p>
-          </li>
-          <li class="p-stepped-list__item u-no-padding--bottom">
-            <p class="p-stepped-list__title">Security configuration (hardening, encryption, PKI)</p>
-          </li>
-          <li class="p-stepped-list__item u-no-padding--bottom">
-            <p class="p-stepped-list__title">Landscape deployment and registration</p>
-          </li>
-        </ol>
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Get your team up and running with Ubuntu</h2>
```

Comment:
This should be wrapped together with `p` in a `p-section--shallow`.
<img width="593" alt="Screenshot 2024-10-02 at 11 18 01 AM" src="https://github.com/user-attachments/assets/aa59f385-b8ac-4954-bb5f-3e10171f1c09">